GitUI/HelperDialogs/FormResetCurrentBranch.Designer.cs

```diff
@@ -180,49 +220,62 @@ private void InitializeComponent()
             this.tableLayoutPanel1.Controls.Add(this._NO_TRANSLATE_BranchInfo, 0, 0);
             this.tableLayoutPanel1.Controls.Add(this.commitSummaryUserControl1, 0, 1);
             this.tableLayoutPanel1.Controls.Add(gbResetType, 0, 2);
-            this.tableLayoutPanel1.Controls.Add(this.flowLayoutPanel1, 0, 3);
+            this.tableLayoutPanel1.Controls.Add(this.linkGitReset, 0, 3);
+            this.tableLayoutPanel1.Controls.Add(this.flowLayoutPanel1, 0, 4);
             this.tableLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.tableLayoutPanel1.Location = new System.Drawing.Point(8, 8);
+            this.tableLayoutPanel1.Location = new System.Drawing.Point(5, 5);
+            this.tableLayoutPanel1.Margin = new System.Windows.Forms.Padding(2);
             this.tableLayoutPanel1.Name = "tableLayoutPanel1";
-            this.tableLayoutPanel1.Padding = new System.Windows.Forms.Padding(4);
-            this.tableLayoutPanel1.RowCount = 4;
+            this.tableLayoutPanel1.Padding = new System.Windows.Forms.Padding(3);
+            this.tableLayoutPanel1.RowCount = 5;
+            this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
-            this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 20F));
-            this.tableLayoutPanel1.Size = new System.Drawing.Size(703, 546);
+            this.tableLayoutPanel1.Size = new System.Drawing.Size(469, 459);
             this.tableLayoutPanel1.TabIndex = 0;
             // 
+            // linkGitReset
+            // 
+            this.linkGitReset.AutoSize = true;
+            this.linkGitReset.Location = new System.Drawing.Point(6, 417);
+            this.linkGitReset.Name = "linkGitReset";
+            this.linkGitReset.Size = new System.Drawing.Size(91, 13);
+            this.linkGitReset.TabIndex = 4;
+            this.linkGitReset.TabStop = true;
+            this.linkGitReset.Text = "More see git-reset";
```

Comment:
I don't really like this label. I would prefer "For more information, see 'git reset' documentation"